docs/tool/GUI.ModsManager/index.md

```diff
@@ -32,7 +32,7 @@ Asset Example:
   - name: files/modified_msn.bar
 ```
 
-* `binarc` - Specifies a modification to a subfile within a binarc, using one of the available methods. See `binarc methods` for details on implementing a specific method.
+* `binarc` (KH2, BBS, DDD) - Specifies a modification to a subfile within a binarc, using one of the available methods. See `binarc methods` for details on implementing a specific method.
```

Comment:
The documentation looks very good! The only very small thing is that `binarc` is only used in KH2.

Reply:
It's what I was going to say. I believe `binarcs` are KH2-specific files.

Reply:
Fixed